geometry_msgs/Point position #Position gerastert nach Map
string richtung #heading Nord Sued West Ost